### DISCUSSION TITLE
need to improve behavior of autocomplete

### DISCUSSION THREAD

**Discussion by girishvisaero** [mantinedev/mantine#4862] [Feature requests]
in the Autocomplete, when I focus on autocomplete, It just suggest me a selected country in the popover, can we improve it like when user focus on it, popover list should have all data list, instead a single controlled value?

**Comment by rtivital** [comment]
As stated in the documentation, Autocomplete is opinionated component – https://mantine.dev/core/autocomplete/#combobox
If it does not fit your requirements, you can build a custom component with Combobox – https://mantine.dev/combobox/?e=BasicAutocomplete
